hoomd/md/pair/pair.py

```diff
@@ -635,9 +629,7 @@ class DPD(Pair):
     utilize the DPD functionality in your work.
 
     `DPD` does not implement and energy shift / smoothing modes due to the
```

Comment:
I'm guessing this is supposed to say "any," but I suppose it might support the default behavior of `"none"`?
```suggestion
    `DPD` does not implement any energy shift / smoothing modes due to the
```